components/composited/AccountPage/AccountPage.tsx

```diff
@@ -0,0 +1,43 @@
+import { useState } from 'react';
+import { useMutation } from 'react-query';
+
+import { updateUser } from '../../../lib/post/updateUser';
+import { Auth } from '../../generic/Auth/Auth';
+import { AccountForm } from '../AccountForm/AccountForm';
+import { PasswordForm } from '../../composited/PasswordForm/PasswordForm';
+import { Heading } from '../../generic/Heading/Heading';
+import { ErrorMessage } from '../../generic/ErrorMessage/ErrorMessage';
+import { SuccessMessage } from '../../generic/SuccessMessage/SuccessMessage';
+
+export const AccountPage = () => {
+  const [updateSuccessMessage, setUpdateSuccessMessage] = useState(false);
```

Comment:
It can be (and should) be read from `react-query` hook state. You can read `isSuccess` field.